DependencyInjection/FOSRestExtension.php

```diff
@@ -230,6 +231,10 @@ private function loadVersioning(array $config, XmlFileLoader $loader, ContainerB
     private function loadParamFetcherListener(array $config, XmlFileLoader $loader, ContainerBuilder $container)
     {
         if ($config['param_fetcher_listener']['enabled']) {
+            if (!class_exists(Constraint::class)) {
+                @trigger_error('The ParamFetcher requirements feature requires the symfony/validator component.', E_USER_DEPRECATED);
```

Comment:
I think we should reword this a bit now:

> Enabling the `fos_rest.param_fetcher_listener` option when the Symfony Validator component is not installed is deprecated since FOSRestBundle 2.6 and will throw an exception in 3.0. Disable the feature or install the `symfony/validator` package.

Reply:
@xabbuh Reworded, thank you :)